package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -306,8 +314,8 @@ def update_dictionary_end_frame(array_simulation_particle_coordinates, dictionar
     return dictionary_cube_data_this_core
 
 
-def produce_coordinate_arrays_single_process(topology_file_path, trajectory_file_path, MDA_selection, start_frame,
-                                             end_frame):
+def produce_coordinate_arrays_single_process(topology_file_path: str, trajectory_file_path: str, MDA_selection: str,
+                                             start_frame: int, end_frame: int) -> Tuple[Sequence[int], Sequence[int]]:
```

Comment:
```suggestion
def produce_coordinate_arrays_single_process(
    topology_file_path: str, trajectory_file_path: str,
    MDA_selection: str,
    start_frame: int, end_frame: int,
) -> Tuple[np.ndarray, np.ndarray]:
```

From what I can see, you got the return type wrong.  Both returned values are coordinate arrays.